ISSUE_TEMPLATE.md

```diff
@@ -1,13 +1,17 @@
-Please make sure that the boxes below are checked before you submit your issue. Only use GitHub issues for issues with the implementation, not for issues with specific datasets or general questions about functionality. If your issue is an implementation question, please ask your question on the [#keras-retinanet Slack channel](https://github.com/fizyr/keras-retinanet#discussions) and ask there instead of filing a GitHub issue.
+Please make sure that you follow the steps below when creating an issue.
+Only use GitHub issues for issues with the implementation, not for issues with specific datasets or general questions about functionality.
+If your issue is an implementation question, please ask your question on the #keras-retinanet Slack channel instead of filing a GitHub issue.
+You can find the Slack channel here: https://github.com/fizyr/keras-retinanet#discussions
```

Comment:
Maybe change this to "You can find directions to the Slack channel here" ?

Reply:
Good point.

Reply:
Updated.

Reply:
Then let's merge after Travis is done.